gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/EhrExtractMessageHandler.java

```diff
@@ -248,6 +263,17 @@ private NACKMessageData prepareNackMessageData(NACKReason reason, RCMRIN030000UK
             .build();
     }
 
+    private String parseNhsNumber(RCMRIN030000UK06Message payload) {
```

Comment:
perhaps these parse methods could be pulled into a util class? (and tested)

Reply:
Have move these into a util class now